pkg/platform/platform.go

```diff
@@ -281,3 +282,36 @@ func WorkerPoolNamesUnique(pools []WorkerPool) hcl.Diagnostics {
 
 	return d
 }
+
+// CheckCPUManagerPolicy takes a map with key as worker pool name and value as the CPU Manager
+// Policy value. This function checks if the values are valid.
+func CheckCPUManagerPolicy(wp map[string]string) hcl.Diagnostics {
```

Comment:
`wp` doesn't seem to me like a clear name here. And function name does not help either :smile: I know the comment, but still :)

Reply:
Modified the name, also added more context to the command.